tests/src/baseservices/typeequivalence/simple/Simple.cs

```diff
@@ -8,7 +8,6 @@
 using System.Reflection;
 using System.Runtime.InteropServices;
 
-using TestLibrary;
```

Comment:
I am pretty sure this is needed for the `Assert` call below - at least that is what I intended when I wrote this test.

Reply:
Ah ok. Then I need to revert changes for this project. 

I tested all the changes on Linux machine and they succeeded. However, this test is built as Windows only, so I didn't catch the error earlier.

Reply:
>I tested all the changes on Linux machine and they succeeded

Oops. Yeah all COM and TypeEquivalence tests are Windows only. There are also some other interop tests that have some tricky behavior on non-Windows - just FYI.